### ISSUE TITLE
Unable to build 3.0-beta.8 due to unavailable mtap Maven  dependency

### ISSUE THREAD

**Issue by DaveJohnson8080** [nlpie/biomedicus#84]
**Describe the bug**
Unable to build 3.0-beta.8 due to unavailable mtap Maven  dependency

**To Reproduce**
python setup.py build

**Expected behavior**

Build to complete with out errors.

**Terminal Output**

```
python setup.py build
running build
running build_py
Starting a Gradle Daemon (subsequent builds will be faster)

FAILURE: Build failed with an exception.

* What went wrong:
Could not determine the dependencies of task ':shadowJar'.
> Could not resolve all dependencies for configuration ':runtimeClasspath'.
   > Could not find any version that matches edu.umn.nlpie:mtap:[0.8.0, ).
     Versions that do not match:
       - 0.7.0
       - 0.6.0
       - 0.5.0
       - 0.4.0
       - 0.3.0
       - + 1 more
     Searched in the following locations:
       - https://repo.maven.apache.org/maven2/edu/umn/nlpie/mtap/maven-metadata.xml
       - file:/home/dave/.m2/repository/edu/umn/nlpie/mtap/
     Required by:
         project :

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 10s
error: Java build failed.

```
**Environment**
 - OS: Ubuntu 18.04
 - Version e.g. 3.0-beta.8
 - Python Version 3.8
 - Java Version openjdk 11.0.9.1

**Additional context**
None

**Comment by benknoll-umn** [comment]
OSSRH dropped a couple of our jars for some reason during publishing of the 0.8.0 build of MTAP. I will publish a fix this week on both MTAP / biomedicus.

(Edit) If you need an immediate fix before I get the new version published you can download MTAP source code (GitHub.com/nlpie/mtap) and run "./gradlew publishToMavenLocal" from the "java" directory in that project.

**Comment by benknoll-umn** [comment]
This should be resolved now with the 0.8.1 version of MTAP that I just pushed to maven central.
